### PR TITLE
Reduce names on PSR index

### PR DESCRIPTION
**Pull request by vdelau**
- Editors become maintainers after PSR is accepted, updated labels
- Move credits of past contributors to meta files, to reduce clutter
- Add meta file for PSR-0 and PSR-1
- Reference new PSR for PSR-0 and PSR-2

See also php-fig/www.php-fig.org#302

### CODE REVIEW

File: accepted/PSR-1-basic-coding-standard-meta.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# PSR-0 Meta Document
```
Comment (shadowhand) [inline accepted/PSR-1-basic-coding-standard-meta.md:1]:
```suggestion
# PSR-1 Meta Document
```

**Review by shadowhand — CHANGES_REQUESTED** [review]

File: index.md [hunk 2]
```diff
@@ -8,30 +8,30 @@ As also described in the [PSR Workflow Bylaw][workflow]. The Editor, or editors,
 
 ### Accepted
 
-| Num | Title                          | Editor                         |  Coordinator            | Sponsor                 |
```
Comment (samdark) [inline index.md:11]: Isn't that historical info we want to keep?

Comment (vdelau) [reply index.md:11]: I will have to make another pass, but the intent is to ensure all information is available in the meta document. I introduced meta documents for PSR's that don't have it.

File: index.md [hunk 3]
```diff
@@ -43,20 +43,20 @@ As also described in the [PSR Workflow Bylaw][workflow]. The Editor, or editors,
...
 |:---:|--------------------------------------|--------------------------------|------------|
-| 0   | [Autoloading Standard][psr0]         | Matthew Weier O'Phinney        | Deprecated |
-| 1   | [Basic Coding Standard][psr1]        | Paul M. Jones                  | Accepted   |
-| 2   | [Coding Style Guide][psr2]           | Paul M. Jones                  | Deprecated |
+| 0   | [Autoloading Standard][psr0]         |                                | Deprecated |
+| 1   | [Basic Coding Standard][psr1]        | _vacant_                       | Accepted   |
+| 2   | [Coding Style Guide][psr2]           |                                | Deprecated |
 | 3   | [Logger Interface][psr3]             | Jordi Boggiano                 | Accepted   |
-| 4   | [Autoloading Standard][psr4]         | Paul M. Jones                  | Accepted   |
+| 4   | [Autoloading Standard][psr4]         | _vacant_                       | Accepted   |
```
Comment (Jean85) [inline index.md:59]: Shouldn't we leave this blank?

**Review by Jean85 — CHANGES_REQUESTED** [review]

**Review by Jean85 — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by Jean85** [comment]
@vdelau can you handle the conflicts?

**Review by ashnazg — APPROVED** [review]
I've cleared the conflicts.

**Review by Jean85 — APPROVED** [review]

**Pull request merged**